README.md

```diff
@@ -1 +1,3 @@
-# GameEngine
\ No newline at end of file
+# GameEngine
```

Comment:
In a different PR, Kirsten changed this to `ffyrn.gameengine` and in an earlier commit Harry changed it to `Ffyrn.MonoEngine`. What are we deciding on?

Reply:
`Ffyrn.MonoEngine` would be consistent with the name of the project & namespace and I believe is the name we agreed on. It should be in PascalCase to follow the .NET naming conventions